public/scripts/Features.jsx

```diff
@@ -162,7 +181,8 @@ const mapStateToProps = (state, ownProps) => {
 
 const ffMapDispatchToProps = dispatch => (
   {
-    groupToggleCheckedFeatures: prefix => dispatch(Action.groupToggleCheckedFeatures(prefix))
+    groupToggleCheckedFeatures: ctgy => dispatch(Action.groupToggleCheckedFeatures(ctgy)),
```

Comment:
Can we `props.dispatch` directly, or do we need to work through DispatchToProps?